documentation/airport-guides/ksea.md

```diff
@@ -0,0 +1,74 @@
+### About this facility
+Seattle TRACON (S46), Seattle, Washington, USA
+582,394 annual operations in 2016 ([FAA ATADS](https://aspm.faa.gov/opsnet/sys/Tracon.asp))
+
+Located less than a quarter mile west of SeaTac's Runway 16R-34L, this consolidated approach control facility works all traffic in the Seattle metropolitan area. The facility is fairly new, with a large, open control floor sporting about 15-20 STARS scopes, and an in-house Traffic Management Unit.
+
+### Airspace
+For the most part, S46 airspace extends up to 15,000 feet over a fairly large area surrounding northwestern Washington state. There are various shelves that extend S46's ability to prepare an arrival sequence, including the Olympia and Arlington shelves (SFC-9,000), and the Paine shelf which extends the northern reach of the airspace.
+
+Class B airspace exists with tops of 10,000 over the Seattle-Tacoma metropolitan area, with a fairly complex structure designed to deconflict noncommunicative VFR traffic from IFR arrivals on arrival and departure procedures in and out of Seattle.
+
+Of unique note here is the location of Boeing Field (KBFI), which is less than 5nm north of SeaTac (KSEA), right along the approach course to the 16s. For this reason, extra restrictions on localizer intercept altitudes or crossing restrictions are needed to ensure appropriate separation. Particularly when in good visual conditions on a south/east flow, due to the numerous conveniently placed landmarks, controllers heavily utilize visual reference points with their traffic advisories.
+
+### Instrument Procedures
+For an American airport, the procedures are fairly automated, with the design of the SIDs and STARs taking a lot of weight off the controllers. Arrivals climb and descend via the procedures, and by design the traffic flow requires minimal intervention other than running the final. Aircraft are still manually vectored to the final approach course.
```

Comment:
> Arrivals climb and descend via the procedures  

Should probably be `Aircraft climb and descend via the procedures`

Reply:
done